docs/chainbridge/setup/setup-polygon-sdk.md

```diff
@@ -0,0 +1,98 @@
+---
+id: setup-polygon-sdk
+title: Polygon SDK Setup
+---
+
+In this section, you will setup Polygon SDK network locally.
+
+First, you will get the codes from the repository, build the codes, and move the binary under /usr/local/bin so that you can run it as a command.
+
+```bash
+$ git clone https://github.com/0xPolygon/polygon-sdk.git
+$ cd polygon-sdk
+$ go build -o main main.go && mv ./main /usr/local/bin/polygon-sdk
+```
+
+## Step 1: Generate validator keys
+
+Next, we'll generate validator keys and genesis.json. In IBFT that is a consensus Polygon SDK has, there are 4 validators to run at least. `ibft init` command creates new keys and prepare for the data directory node will store data into. `ibft init` commands will output validator address and Libp2p ID.
+
+```bash
+# Setup keys and prepare for data directory for validators
+$ polygon-sdk ibft init --data-dir test-chain-1
+
+[IBFT INIT]
+Public key (address) = <Node 1 Validator Address>
+Node ID              = <Node 1 Libp2p ID>
+
+$ polygon-sdk ibft init --data-dir test-chain-2
+
+[IBFT INIT]
+Public key (address) = <Node 2 Validator Address>
+Node ID              = <Node 2 Libp2p ID>
+
+$ polygon-sdk ibft init --data-dir test-chain-3
+
+[IBFT INIT]
+Public key (address) = <Node 3 Validator Address>
+Node ID              = <Node 3 Libp2p Node ID>
+
+$ polygon-sdk ibft init --data-dir test-chain-4
+
+[IBFT INIT]
+Public key (address) = <Node 4 Validator Address>
+Node ID              = <Node 4 Libp2p Node ID>
+```
```

Comment:
Remove this, and refer to the existing local setup docs, or even better say what are the prerequisites for following this guide, e.g. a working Polygon-SDK network with the following addresses premined.

Reply:
Removed at 5261fb94e15d56179ba90ddf76cff162ba02e03a